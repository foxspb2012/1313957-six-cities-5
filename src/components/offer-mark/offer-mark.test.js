import React from 'react';
import OfferMark from './offer-mark';
import renderer from 'react-test-renderer';

const Type = {
  OFFER_PAGE_CONTENT_CARD: `place-card`,
  OFFER_CARD: `property`,
};

describe(`Should OfferMark render correctly`, () => {
  it(`Should OfferMark render correctly offer-page-content card typeClass`, () => {
    const tree = renderer
      .create(
          <OfferMark typeClass={Type.OFFER_PAGE_CONTENT_CARD} />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`Should OfferMark render correctly offer card typeClass`, () => {
    const tree = renderer
      .create(
          <OfferMark typeClass={Type.OFFER_CARD} />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
