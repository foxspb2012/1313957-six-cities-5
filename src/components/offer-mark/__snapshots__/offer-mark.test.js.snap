// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should OfferMark render correctly Should OfferMark render correctly offer card typeClass 1`] = `
<div
  className="property__mark"
>
  <span>
    Premium
  </span>
</div>
`;

exports[`Should OfferMark render correctly Should OfferMark render correctly offer-page-content card typeClass 1`] = `
<div
  className="place-card__mark"
>
  <span>
    Premium
  </span>
</div>
`;
